### ISSUE TITLE
Youtube API update

### ISSUE THREAD

**Issue by Iridahonda** [Duell10111/ReactTube#13]
Need to change package.json to the latest youtube.js to fix an YT API update: 

"youtubei.js": "https://github.com/LuanRT/YouTube.js.git#main"

**Comment by Duell10111** [comment]
Good point, they made a new release today, therefore I will create a PR for the new version. :)

**Comment by Iridahonda** [comment]
Yup - seems ok.  Cheers!
